README.md

```diff
@@ -164,8 +178,7 @@ and to recipients of software distributed by American Express, You reserve all
 right, title, and interest, if any, in and to Your Contributions. Please [fill
 out the Agreement](https://cla-assistant.io/americanexpress/xcprojectlint).
 
-Please feel free to open pull requests and see
-[CONTRIBUTING.md](./CONTRIBUTING.md) for commit formatting details.
+Please feel free to open pull requests.
```

Comment:
Why remove the link to Contributing?

Reply:
That file doesn't exist.

Reply:
TIL: `CONTRIBUTING.md` is gone

Reply:
Yep!  I noticed that early on when looking for the formatting details.  In the end, I just went with how we currently format our code and used that on the files I touched.